README.md

```diff
@@ -1,8 +1,60 @@
+
+**Competent:** C, JavaScript, Python, HTML, CSS
+
+**Familiar:** Docker, GCP, Pub/Sub, REST API, Spring, Django/Flask, Matlab
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AnneLi](https://github.com/jarviscanada/jarvis_data_eng_AnneLi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/linux_sql)]: This project automates the process of monitoring servers/nodes of linux that are connected through a switch and communicates internally through IPv4 addresses. Implemented bash scripts that obtain data on hardware specifications and on-going Linux resource usage data.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/core_java)]:
+      
+  - Twitter App: Implemented a CLI app that post|show|delete tweets to the user account from the command-line via Twiiter REST API using the MVC architecture.
+  - JDBC App: This project sets up a connection to the PostgreSQL database by leveraging the JDBC API that manages customer and order records.
+  - Grep App: This app echoes partial functionality of the linux command `grep`. The app searches for a text pattern recursively through all the files in a given directory and outputs the matched lines to a designated file.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/springboot)]: In-progress
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/python_data_anlytics)]: In-progress
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/spark)]: In-progress
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/cloud_devops)]: In-progress
+
+
+## Highlighted Projects
+**SERVE.AI (Web application)**: Developed an Online Cafeteria Application utilizing the Django framework, SQLite3, Bootstrap using Agile methodologies and practices. Practiced SDLC process models, requirements elicitation, system modelling, project management, teamwork, navigating documentations and software testing.
+
+**SMART CITY APPLICATION**: Learned about distributed System and networking technology by designing and developing an urban IoT website which generates pickup schedule based on real-time capacity of sensor-attached garbage bin. Programmed ESP8266 using C++ in Arduino IDE to send JSON data via MQTT protocol to Raspberry Pi.
+
+
+## Professional Experiences
+
+**Data Engineer, Jarvis (Jan 2021-present)**: Implemented projects that solves Big-data problems utilizing a set of tools and technologies such as Linux, SQL, Core Java, SpringBoot, DevOps/Cloud, Hadoop and Spark/Scala. Conducted and participated in daily scrum meetings in a bi-weekly sprint.
```

Comment:
Conducted and participated in daily scrum meetings in a bi-weekly sprint. _> talk more about agile